src/server/scripts/EasternKingdoms/Deadmines/deadmines.h

```diff
@@ -23,6 +23,8 @@
 #define DMScriptName "instance_deadmines"
 #define DataHeader "DM"
 
+uint32 const EncounterCount     = 7;
```

Comment:
could this be constexpr ?

Reply:
I am not sure what you mean here

Reply:
change `uint32 const` to `uint32 constexpr`, this is a compile-time constant

Reply:
Is that new? In a few other dungeon/raid files this is not used either:
https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/scripts/Outland/BlackTemple/black_temple.h#L26
https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/scripts/Northrend/Naxxramas/naxxramas.h#L26

Reply:
It's a recent addition to the C++ language

Reply:
I will change it, but I am not sure what the advantage of  `constexpr` over the `const` is.